.github/automation/clang-format.sh

```diff
@@ -16,7 +17,7 @@
 # limitations under the License.
 #===============================================================================
 
-CLANG_FORMAT=clang-format-11
+CLANG_FORMAT="${CLANG_FORMAT:-"clang-format-11"}"
```

Comment:
The plan here was to use clang-format-11 unconditionally for consistency. Why do we want to take a value from the environment?